src/sqlfluff/dialects/dialect_athena.py

```diff
@@ -643,3 +644,31 @@ class IntervalExpressionSegment(BaseSegment):
             ),
         ),
     )
+
+
+class ShowStatementSegment(BaseSegment):
+    """A `show` execute statement.
+
+    Full Apache Hive `SHOW` reference:
+    https://cwiki.apache.org/confluence/display/Hive/LanguageManual+DDL#LanguageManualDDL-Show
+
+    Athena supported subset:
+    https://docs.aws.amazon.com/athena/latest/ug/ddl-reference.html
+    """
+
+    type = "show_statement"
+    match_grammar = Sequence(
+        "SHOW",
+        OneOf(
+            Sequence(
+                "TABLES",
+                Sequence("IN", Ref("TableReferenceSegment"), optional=True),
+                Ref("ExpressionSegment", optional=True),
+            ),
+            Sequence(
+                "VIEWS",
+                Sequence("IN", Ref("TableReferenceSegment"), optional=True),
+                Sequence("LIKE", Ref("ExpressionSegment"), optional=True),
```

Comment:
so i'm mostly concerned about this approach - i'm currently trying to address just my use case, but if one were to fully cover the show keyword in Athena, there would be 8 nested bespoke clauses that are all based off the same keyword. should these all be in the same segment? or should they be broken out a bit more?

Reply:
I would say for this, it's fine to have all of these wrapped in a big OneOf. Mainly because the individual statements are so small for each type. It's primarily just about style preference at this point, and I don't have one unless one style becomes significantly less performant than another.

Reply:
ok - so thinking about the next steps in this area, if i'm gonna start walking down DBs that support these (i'm sure of hive & mysql, but i could poke around in some of the others), would it be better to do one PR per DB type, or just try to hit things in one broad stroke?